Disable most menu items if user isn't logged in

codebykat opened the pull request:
### Fix

Most menu items aren't applicable when logged out, and don't work anyhow, so let's hide them.

We could also gray them out instead of remove them entirely; anyone have a preference?

### Test

1. Open the app, signed out
2. Note that irrelevant menu options do not appear (Format menu, Preferences, etc)
3. Log in
4. Note that all menu items are now visible and work as intended
5. Log out again
6. Note that menu items are once again disabled

### Release

Not updated: Disable inapplicable menu items when logged out

codebykat committed:
working except for logout

codebykat committed:
remove unused import

codebykat committed:
reset on logout

codebykat committed:
linting

belcherj commented:
I think we should also remove:
- File -> Print
- View:
  - Zooms
  - Theme

I am going to approve as this works great and code looks good.

belcherj reviewed (approved):
Ship

codebykat committed:
remove zoom and theme when logged out

codebykat committed:
remove spellcheck toggle when logged out

codebykat committed:
remove print from logged out menu and rejigger preferences and file

codebykat commented:
I removed Print (I suppose there's no valid reason to want to print the login screen).

I also removed Zoom and Theme; I feel like these should work on the login screen, but at the moment they don't, so let's remove them for now. Ditto for the spellcheck toggle; maybe it should work, but at the moment, it doesn't, so out it goes.

The other big elephant in the room is About, which I'm going to try to get working in a follow-up PR.

codebykat commented:
Just to note if we'd rather gray these out instead of remove them it's easily enough done by using `enabled` instead of `visible`.

https://github.com/Automattic/simplenote-electron/issues/2137#issuecomment-643301865
